ReactWindows/ReactNative.Net46/Tracing/NullLoggingActivityBuilder.cs

```diff
@@ -15,8 +15,13 @@ public IDisposable Start()
         {
             return new NullDisposable();
         }
+
+        public NullDisposable Create()
```

Comment:
>NullDisposable [](start = 15, length = 14)

What component is using the public API?  I really would still prefer to avoid nested public classes if possible.  If you need the NullDisposable elsewhere, Maybe move it into a helper file?